src/test/resources/fixtures/hmh-ri/caliperEventAssessmentStarted.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "type":"AssessmentEvent",
+  "id":"urn:uuid:uniqueIdPerEvent",
+  "actor":{
+    "id":"https://https://www.hmhco.com/ed/users/userRefId",
+    "type":"Person",
+    "name" : "Casandra Rath"
+  },
+  "action":"Started",
+  "object":{
+    "id":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId",
+    "type":"Assessment",
+    "name":"HMH-RI Quiz Name - Reading Inventory",
+    "dateToStartOn":"2016-11-14T05:00:00.000Z",
+    "dateToSubmit":"2016-11-18T11:59:59.000Z",
+    "maxAttempts":1,
+    "maxSubmits":1,
+    "maxScore":25.0,
+    "version":"1.0"
+  },
+  "generated":{
+    "id":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId/attempts/1",
+    "type":"Attempt",
+    "assignee":"https://https://www.hmhco.com/ed/users/userRefId", (same as actors id)
+    "assignable":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId", (same as object id)
+    "count":1,
+    "dateCreated":"2016-11-15T10:15:00.000Z",
```

Comment:
The dateCreated is the time the attempt was created, which is also just now.

Reply:
In caliper you get one entry for a group, which is an Organization. Most commonly that group is a CourseSection (which is a class), however it can also be a true group, i.e. a named list of members.
A lot of the new functionality for assessment in Ed for Renaissance Learning is assigning users to groups for assessments, so it seems like a good idea to me to represent the group in the Caliper event.
You don't have to lose the class, if the group is a sub set of the class then the group can be a subOrganization of the CourseSection (group).

Reply:
I understand what you are saying - and if there was no group of students within the class, this group would fall back to being just the CourseSection itself right?

Reply:
Exactly